src/mapping/tests/AxialGeoMultiscaleMappingTest.cpp

```diff
@@ -19,9 +19,9 @@ using namespace precice::mesh;
 BOOST_AUTO_TEST_SUITE(MappingTests)
 BOOST_AUTO_TEST_SUITE(AxialGeoMultiscaleMapping)
 
-BOOST_AUTO_TEST_CASE(testConsistentSpread)
+BOOST_AUTO_TEST_CASE(testConsistentSpreadX)
 {
-  /*  The following test works by creating two dimensionally heterogeneous meshes, namely 1D and 3D.
+  /*  The following test works by creating two dimensionally heterogeneous meshes, namely 1D and 3D, coupled along the x-axis.
```

Comment:
Is the "coupled along the x-axis" true?

Reply:
Yes, it is. This is another test.

Reply:
Yes I think so, in the sense that the circular interface is in the z-y-plane.
![image](https://github.com/precice/precice/assets/99761626/796433b4-0f89-4b11-a7e3-125bfe269742)
I had to change a coordinate because it was not explicitly along the x-axis previously.